docs/src/main/asciidoc/deploying-native-executable-openshift.adoc

```diff
@@ -0,0 +1,196 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+
+[id="deploy-openshift-native-executable"]
+= Deploy a Quarkus native application on OpenShift
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud,native
+
+Deploy native Quarkus applications to OpenShift by using the Docker build strategy.
+
+The `quarkus-container-image-openshift` extension uploads a native binary and the Dockerfile to OpenShift.
+Then, OpenShift performs the build to create the container image.
+
+To use this strategy, you must build a native executable on your machine that matches the OpenShift architecture.
+First, it needs to be a Linux executable.
+Then, this executable must match the architecture used by your OpenShift nodes.
+In general, `linux/amd64` executables work.
+
+Your Quarkus project includes pre-generated Dockerfiles with instructions.
+To use a custom Dockerfile, add the file in the `src/main/docker` directory or anywhere inside the module, and set the path to your Dockerfile by using the `quarkus.openshift.native-dockerfile` property.
+
+For information about the Docker build strategy, see xref:deployment-to-openshift.adoc#overviewbuildstrategies[Quarkus deployment to OpenShift].
+
+
+== Prerequisites
+:prerequisites-no-graalvm:
+include::{includes}/prerequisites.adoc[]
+* Access to an OpenShift cluster and the latest compatible version of the `oc` tool installed
+For more information, see xref:openshift-login-cluster-reference.adoc[OpenShift cluster login]
+* Optional: Access to the link:https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI] (Optional as required only for manual deployment)
```

Comment:
The `oc` tool is already mentioned in previous list item. Also, since the rest of the guide describes a scenario which uses the `oc` command-line tool, I'd say it's not optional.

Or, alternatively, the guide should describe a way to execute the the scenario without `oc`. An incomplete example of this is shown in https://quarkus.io/version/main/guides/deploying-to-openshift#log-into-the-openshift-cluster.